pybamm/experiments/experiment.py

```diff
@@ -51,7 +63,12 @@ def __init__(
         parameters=None,
         period="1 minute",
         use_simulation_setup_type="new",
+        drive_cycles={},
     ):
+
+        import numpy as np
+        self._np = np
```

Comment:
I've never seen this style of import before. Usually we just import modules globally at the top of the file. Is there a style guide somewhere that says this is a better way of doing it?

Reply:
This was actually a style issue.